### PR TITLE
Added `useValue` tests.

### LINKED ISSUE #1055: Enhance Test Coverage for `@yamada-ui/use-value`
package: @yamada-ui/use-value

### Description
`@yamada-ui/use-value` has a test coverage of less than 80%. We will enhance it to above 80%.

### Target files
- [packages/hooks/use-value/src/index.ts](https://app.codecov.io/gh/yamada-ui/yamada-ui/blob/main/packages/hooks/use-value/src/index.ts)
  - [L24](https://app.codecov.io/gh/yamada-ui/yamada-ui/blob/main/packages/hooks/use-value/src/index.ts#L24)
  - [L34](https://app.codecov.io/gh/yamada-ui/yamada-ui/blob/main/packages/hooks/use-value/src/index.ts#L34)

### Problem Statement/Justification
Testing is essential to ensure quality when operating a package.

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.

### PR DESCRIPTION
**Pull request by hirotomoyamada**
Closes #1055

## Description

Added `useValue` tests.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: packages/test/src/index.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
-// @ts-ignore
+import type MatchMediaMock from "vitest-matchmedia-mock"
+export { MatchMediaMock }
```
Comment (nakayan5) [inline packages/test/src/index.ts:2]: @hirotomoyamada 
are these required?

File: packages/test/src/mocks/match-media.ts [hunk 2]
```diff
@@ -1,20 +1,5 @@
...
-        removeListener: vi.fn(),
-        dispatchEvent: vi.fn(),
-      }
-    },
-  }
+import MatchMediaMock from "vitest-matchmedia-mock"
 
-  Object.defineProperty(window, "matchMedia", desc)
-}
+export { MatchMediaMock }
```
Comment (nakayan5) [inline packages/test/src/mocks/match-media.ts:3]: @hirotomoyamada 
need to export this ?

Comment (hirotomoyamada) [reply packages/test/src/mocks/match-media.ts]: @nakayan5 
 
 Delete line.

File: packages/test/src/index.ts (hunk 1 above)
Comment (hirotomoyamada) [reply packages/test/src/index.ts:2]: @nakayan5 

This is required for type definitions in external source code.

**Review by nakayan5 — APPROVED** [review]

**Pull request merged**